### PR TITLE
Adding Adaptive RI profile

### PR DESCRIPTION
**Pull request by idavis**

### CODE REVIEW

**Review by cesarzc — APPROVED** [review]

File: pip/qsharp/_native.pyi [hunk 1]
```diff
@@ -4,6 +4,14 @@
 from enum import Enum
 from typing import Any, Callable, ClassVar, Optional, Dict, List
 
+class TargetCapabilities(Enum):
```
Comment (swernli) [inline pip/qsharp/_native.pyi:7]: Are these used anywhere yet? It seems like they are aren't.

**Review by swernli — APPROVED** [review]

File: pip/qsharp/_native.pyi (hunk 1 above)
Comment (idavis) [reply pip/qsharp/_native.pyi:7]: Good catch, this slipped through the cracks.

**Comment by minestarks** [comment]
How will the user know to associate the phrase "Adaptive RI" with the Quantinuum target?

**Review by minestarks — APPROVED** [review]
LGTM as long as we figure out what to do with the documentation.

**Pull request merged**